### ISSUE TITLE
Error while running script

### ISSUE THREAD

**Issue by NilaWaf3r** [carpikes/ds4-tools#34]
Hello, could you help me in solving the issue I'm having? I keep getting "[Errno None] b'libusb0-dll:err [claim_interface] could not claim interface 0, win error: The parameter is incorrect.\r\n'" whenever I use analog-center or analog range. Just to add I have followed all steps and have installed the libusb driver for my dualsense and the firmware on the controller is 0458. Thank you!

**Comment by NilaWaf3r** [comment]
![image](https://github.com/carpikes/ds4-tools/assets/166095526/d3377fe1-e01f-469b-a78d-36a4da227d2e)

**Comment by nuadaxxx** [comment]
Reinstall again libusb-win32 (v1.2.7.3)

**Comment by carpikes** [comment]
Thanks @nuadaxxx.
Also, I'm working on a web-ui that does not require any driver to be installed (on windows it works with the default driver). 
It's still in beta, but it should already be able to calibrate DS4 and DualSense. Give it a try and let me know: https://the.al/dualshock

**Comment by nuadaxxx** [comment]
> Thanks @nuadaxxx. Also, I'm working on a web-ui that does not require any driver to be installed (on windows it works with the default driver). It's still in beta, but it should already be able to calibrate DS4 and DualSense. Give it a try and let me know: https://the.al/dualshock

Yes, works nice... but an option for only left or right range would be nice..

**Comment by carpikes** [comment]
It's not feasible: the calibration algorithm is written in the dualsense. The script (or the web-ui) just tells the controller to perform calibration

**Comment by nuadaxxx** [comment]
> It's not feasible: the calibration algorithm is written in the dualsense. The script (or the web-ui) just tells the controller to perform calibration

Ah ok, the problem exists only with Hall Sticks, but if I calibrate the center manual (with the holes on the Stick), and then use your tool, then it's okay... not sure why, but it seems, that the initial center is also importend.

With the Calibration board from "jun zeng" it's possible to Calibrate the inner and outer range of the stick, maybe something like that exist also in the dual sense calibration process, but that is just a theorie..... but thx for the tool....

**Comment by NilaWaf3r** [comment]
Is there any possibility of downgrading the firmware of the dualsense controller? Ever since updating to the 0458 firmware my right analog has major drift to the left when prior to updating it was completely fine. Also, when slowly pushing to the right it goes left as well.

**Comment by nuadaxxx** [comment]
Normal that drift issue is from bad soldering, I think a downgrade is not possible.

**Comment by carpikes** [comment]
@NilaWaf3r AFAIK not at the moment
